src/features/cards/organisms/card-item.js

```diff
@@ -76,3 +86,48 @@ const CardBox = styled(Card)`
   max-height: 24rem;
   overflow-y: hidden;
 `
+
+const PopUpButton = () => {
+  const [popup, setPopup] = useState(false)
+
+  return (
+    <div>
+      <Button
+        small
+        onClick={() => {
+          setPopup(() => !popup)
+        }}
+      >
+        PopUp
+      </Button>
+      {!!popup && (
+        <div>
+          <PopUp
+            onClose={() => {
+              setPopup(() => false)
+            }}
+          >
+            <Box popup>
+              <Button
+                small
+                onClick={() => {
+                  setPopup(() => false)
+                }}
+              >
+                CLOSE (should be x icon in the corner)
+              </Button>
+
+              <H3>Do you want to delete?</H3>
+              <p>
+                Do you absolutely sure you want to delete? Just kidding we are
+                archiving them anyway.
+              </p>
+              <Button onClick={() => true}>Yes</Button>
```

Comment:
Just call `props.onDeleteClick`

Reply:
I am waiting for API with a real delete function, and alerts are blocked by husky, so this is a temporary dummy. Could I keep it?

Reply:
Use defaultProps